ui/app/AppLayouts/Onboarding2/pages/LoginScreen.qml

```diff
@@ -217,13 +219,15 @@ OnboardingPage {
                        passwordBox.forceActiveFocus()
                     }
 
-                    if (!biometricsAvailable || !isBiometricsLogin)
-                        return
+                    Qt.callLater(() => {
```

Comment:
Why the `callLater`?

Reply:
because we are in `onSelectedProfileKeyIdChanged` and the `isBiometricsLogin` may depend also on `selectedProfileKeyId`. So we need to give a chance for this property to be re-evaluated before we actually read it.